lib/postzord/receiver/local_batch.rb

```diff
@@ -25,6 +25,15 @@ def receive!
     # 09/27/11 this is slow
     notify_users
 
+    if @object.respond_to?(:type) && @object.type == "Reshare"
+      root_object = Post.find_by guid: @object.root_guid
+      root_author = Person.find(root_object.author_id)
```

Comment:
I use ditto to say same as last comment. If something is ok I don't comment on it at all.